fc/cli/main.py

```diff
@@ -29,3 +29,35 @@ def task(title, description, parent_story, username, in_progress, no_assign):
             click.echo('Triage task put into In Progress')
     except HTTPError as exception:
         click.echo('{} task creation failed with {}'.format(new_task.type_str(), exception))
+
+@cli.command()
+def triagesearch():
```

Comment:
I think we need to do sub commands for this and the other command.  Something like...
```bash
$ fcli tasks add "title" "description" [parent story]
$ fcli tasks view
$ fcli tasks score
```